### DISCUSSION TITLE
Locking Pin Thread Parameters

### DISCUSSION THREAD

**Discussion by matthewdominick** [aderusha/DDD-Printable-Wall-Control-System#18] [Q&A] [answered]
I want to make a longer locking pin. What are the thread parameters? 1/2 inch diameter with ANSI threads? If so, which type? For example ANSI 1/2-14 UNS or ANSI 1/2-12 UNS?

**Comment by aderusha** [comment, marked as answer]
Sadly, the answer is "none of the above".  I originally started with utilizing common thread geometry but nothing I had tried worked well when 3D printed, particularly in the short distances involved.  The result is that the thread was custom designed for 3D print and use in this project.

Here's how that worked out:
Pitch: 2.5mm
Crest: .5mm
Minor diameter: 13.5mm
Major diameter: 16mm

I've just added [an STL of the thread model](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Accessories/Locking%20Screw%20thread%20model.stl) to the repo if that'll help you out.  It's 10mm tall, you can copy/paste it to make it as tall as you need.

**Reply by matthewdominick** [reply]
Awesome. Thank you!

**Reply by aderusha** [reply]
@matthewdominick - For whatever reason I was looking at this link again and realized I screwed up this screw!  The model I had originally linked was missing the valley in the thread.  The STL linked above has now been updated with the correct model.  Hope this doesn't catch you too late!
